src/app/modules/main/chat_section/chat_content/messages/module.nim

```diff
@@ -766,3 +781,38 @@ method onFirstUnseenMessageLoaded*(self: Module, messageId: string) =
   self.firstUnseenMessageState.initialized = true
   self.firstUnseenMessageState.fetching = false
   self.reevaluateViewLoadingState()
+
+method forceLinkPreviewsLocalData*(self: Module, messageId: string) =
+  let item = self.view.model().getItemWithMessageId(messageId)
+  debug "forceLinkPreviewsLocalData", messageId, itemFound = $(item != nil)
+  if item == nil:
+    return
+  if not item.linkPreviewModel.updateForcedLocalDataTimestamp():
+    return
+  self.updateLinkPreviewsContacts(item, true)
+  self.updateLinkPreviewsCommunities(item, true)
```

Comment:
Could you add the name of the parameter here for easier readability. I think the name is `seen = true`?